joomla/joomla-cms
# [4.4] Adminmenu: Fix export for child items

Pull Request for Issue #37273 .

### Summary of Changes
When exporting a menu preset in J4, the information about the tree of the menu items gets lost. This fixes that.


### Testing Instructions
1. Go to menu manager, filter by admin and create a new menu for the administrator. Make sure to select a preset to pre-populate the menu.
2. In the menu manager, select the menu and click "Download as Preset"

### Actual result BEFORE applying this Pull Request
You get a XML in return and all `<menuitem>` tags are on the same level.


### Expected result AFTER applying this Pull Request
The menuitem tags are nested properly in the way they are in the menu tree.


### Link to documentations
Please select:
